tests/integration_tests/releases.py

```diff
@@ -98,5 +98,7 @@ def from_os_image(
 MANTIC = Release("ubuntu", "mantic", "23.10")
 NOBLE = Release("ubuntu", "noble", "24.04")
 
+UBUNTU_STABLE = (BIONIC, FOCAL, JAMMY, MANTIC)
```

Comment:
Nit: There's no reason to include bionic here as it is EOL.

Reply:
drop it like it's hawwwt

Reply:
squash merged this comment leaving BIONIC out of UBUNTU_STABLE. Will leave dropping all BIONIC integration tests to a separate PR.